scripting/include/srccoop/manager.inc

```diff
@@ -186,6 +186,18 @@ methodmap CoopManager
 			Timer_DecrementUntilMapStart(null);
 		}
 	}
+	
+	public static void OnPlayerRunCmd(CBasePlayer pPlayer)
+	{
+		if (!data.m_bStarted)
+		{
+			CBaseCombatWeapon pWeapon = pPlayer.GetActiveWeapon();
+			if (pWeapon != NULL_CBASEENTITY)
+			{
+				PreventWeaponFiring(pWeapon, 1.5);
```

Comment:
1.5 is too long, some maps you want to start shooting asap.
OnPlayerRunCmd runs about each 0.01 ~ 0.05 seconds.

Reply:
Moved this value from `1.5` down to `0.25`.